src/v/config/configuration.cc

```diff
@@ -618,19 +618,7 @@ configuration::configuration()
       "wasn't reached",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       1ms)
-  , fetch_read_strategy(
-      *this,
-      "fetch_read_strategy",
-      "The strategy used to fulfill fetch requests",
-      {.needs_restart = needs_restart::no,
-       .example = model::fetch_read_strategy_to_string(
-         model::fetch_read_strategy::non_polling),
-       .visibility = visibility::tunable},
-      model::fetch_read_strategy::non_polling,
-      {
-        model::fetch_read_strategy::polling,
-        model::fetch_read_strategy::non_polling,
-      })
+  , fetch_read_strategy(*this, "fetch_read_strategy")
```

Comment:
@StephanDollberg i would guess that it means upgraded cluster would have no choice but to use the new strategy. what were you thinking would cause the cluster to become useless? did the other plans have different semantics from client perspective?

Reply:
Sorry wasn't clear here, what I mean was something along the lines of "unknown property value, won't start" kind of thing

Reply:
ahh, got it. my understanding was that the deprecated_property would accept and ignore any new changes. definitely good question and i'm not 100% sure.

Reply:
There is a ducktape test that explicitly sets the configuration property marked deprecated in this PR. Since that test is still passing I'm assuming that the values it's being set to are just ignored. I'll look at the code to verify though.

Reply:
Yeah, deprecated properties just ignore any value they're set to.